src/redux/reducers/componentConfig/initialState.js

```diff
@@ -110,110 +146,155 @@ const volcanoInitialConfig = {
-  masterColour: '#000000',
-  titleText: '',
-  titleSize: 20,
-  titleAnchor: 'start',
   axisTitlesize: 13,
-  axisTicks: 13,
-  transGrid: 0,
-  axesOffset: 10,
-  masterFont: 'sans-serif',
-  xaxisText: 'Sample',
-  yaxisText: 'Proportion',
-  pointStyle: 'circle',
-  pointOpa: 5,
-  g1Color: 'red',
-  g2mColor: 'green',
-  sColor: 'blue',
-  legendTextColor: '#FFFFFF',
   geneexpLegendloc: '',
-  labelSize: 28,
-  labelShow: 1,
-  labelFont: 2,
-  labelsEnabled: true,
-  testVar: null,
 };
 
-// This is an updated spec that is only used in data processing and not in plots and tables.
-const dataProcessingEmbeddingCategorical = {
-  spec: '2.0.0',
+// EMBEDDING PREVIEW - Coloured by sample
+const embeddingPreviewBySampleInitialConfig = {
+  spec: '1.0.0',
+  legend: {
+    ...legendBaseState,
+    position: 'top',
+  },
   dimensions: {
+    ...dimensionsBaseState,
     width: 700,
     height: 550,
   },
   axes: {
-    titleSize: 13,
-    fontSize: 13,
-    gridOpacity: 0,
-    gridWidth: 10,
+    ...axesBaseState,
+    xAxisText: 'UMAP 1',
+    yAxisText: 'UMAP 2',
     offset: 10,
-    xText: 'UMAP 1',
-    yText: 'UMAP 2',
-    lineWidth: 2,
   },
   title: {
-    text: '',
-    size: 20,
-    anchor: 'start',
-  },
-  labels: {
-    show: true,
-    size: 28,
+    ...titleBaseState,
+    fontSize: 20,
   },
+  fontStyle: fontStyleBaseState,
+  colour: colourBaseState,
+  marker: markerBaseState,
+  label: labelBaseState,
+  selectedCellSet: 'louvain',
+  selectedSample: 'All',
+};
+
+// EMBEDDING PREVIEW - Coloured by cell sets
+const embeddingPreviewByCellSetsInitialConfig = {
+  spec: '1.0.0',
   legend: {
-    show: true,
+    ...legendBaseState,
     position: 'top',
   },
-  markers: {
-    size: 5,
-    opacity: 5,
-    shape: 'circle',
+  dimensions: {
+    ...dimensionsBaseState,
+    width: 700,
+    height: 550,
+  },
+  axes: {
+    ...axesBaseState,
+    xAxisText: 'UMAP 1',
+    yAxisText: 'UMAP 2',
+    offset: 10,
+  },
+  title: {
+    ...titleBaseState,
+    fontSize: 20,
+  },
+  fontStyle: fontStyleBaseState,
+  colour: colourBaseState,
+  marker: markerBaseState,
+  label: labelBaseState,
+  selectedCellSet: 'louvain',
+  selectedSample: 'All',
+};
+
+// EMBEDDING PREVIEW - Config for fraction of mitochondrial reads
+const embeddingPreviewMitochondrialReadsInitialConfig = {
+  spec: '1.0.0',
+  legend: legendBaseState,
+  dimensions: {
+    ...dimensionsBaseState,
+    width: 700,
+    height: 550,
+  },
+  axes: {
+    ...axesBaseState,
+    xAxisText: 'UMAP 1',
+    yAxisText: 'UMAP 2',
+    domainWidth: 2,
+    offset: 10,
   },
-  text: {
-    font: 'sans-serif',
-    color: '#000000',
+  title: {
+    ...titleBaseState,
+    dx: 0,
+    fontSize: 20,
   },
-  colorInverted: false,
-  data: {
-    cellSet: 'louvain',
+  fontStyle: fontStyleBaseState,
+  colour: colourBaseState,
+  marker: markerBaseState,
+  label: labelBaseState,
+  logEquation: 'datum.expression*1',
+  shownGene: 'CST3',
```

Comment:
for now this is fine, since the embedding is showing mocked data, but when we implement the actual mitochondrial reads, we will need to remove this `showGene` parameter. Similarly for the doublet score view

Reply:
fixed in new commit